recipes/lrphase/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "LRphase" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2375aeb05cac517adb6fe6e9b02574ea98efd298586dfd54a7929c1d8605c60f
+
+build:
+  number: 0
+  entry_points:
+    - LRphase = LRphase.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - biopython >=1.78
+    - numpy >=1.20.1
+    - pip
+    - powerlaw >=1.4.6
+    - pyliftover >=0.4
+    - pysam >=0.16.0.1
+    - powerlaw>=1.4.6
+    - requests>=2.26.0
+    - python
+  run:
+    - biopython >=1.78
+    - numpy >=1.20.1
+    - powerlaw >=1.4.6
+    - pyliftover >=0.4
+    - pysam >=0.16.0.1
+    - powerlaw>=1.4.6
+    - requests>=2.26.0
+    - python
+
+test:
+  imports:
+    - LRphase
+  commands:
+    - LRphase --help
+
+about:
+  home: "https://github.com/Boyle-Lab/LRphase.git"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Phasing individual long reads using known haplotype information."
+  doc_url: 
```

Comment:
Please delete from `doc_url:` to the end of the file.

Reply:
Made the changes you asked for. All should be ready to go now.